src/renderer.ts

```diff
@@ -51,13 +49,14 @@ export class Renderer {
             coverageLines.none = [];
             coverageLines.partial = [];
 
-            // find best scoring section editor combo (or undefined if too low score)
-            const topSection = this.topSectionFinder.findTopSectionForEditor(textEditor, sections);
+            if (!textEditor || !textEditor.document) {return; }
 
-            if (!topSection) { return ; }
+            const section = sections.get(normalizeFileName(textEditor.document.fileName));
 
-            this.filterCoverage(topSection, coverageLines);
-            this.setDecorationsForEditor(textEditor, coverageLines);
+            if (section) {
```

Comment:
@ryanluker this can cause the wrong score, etc. to carry over because it doesn't flush properly. noticed with the Python tests with the score filtering thing.

Reply:
good catch, fixed this in my other PR #152